css-grid_place-elements/css/styles.css

```diff
@@ -68,24 +68,38 @@ body {
 
 .element-1 {
   /* Add styles here */
+  grid-column-start: 3;
+  grid-column-end: 5;
+
 }
 
 .element-2 {
   /* Add styles here */
+  grid-column-start: 5; 
+  grid-column-end: 11;
 }
```

Comment:
When the element goes until the end of the grid, you can use -1, so you don't have to count: grid-column: 5/-1.

Reply:
Very nice tip. Thank you. WIll use it!